config/prod/iwanna-batch-job.yaml

```diff
@@ -0,0 +1,25 @@
+template_path: remote/managed-batch.yaml
+stack_name: iwanna-batch-job
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "joe.smith@sagebase.org"
+  # The docker command to run
+  ContainerCommands: "['echo', 'Hello World']"
+  # The VPC to run batch instances in (do not change)
+  VpcName: "sandcastlevpc"
+  # (Optional) The docker image to run (default 137112412989.dkr.ecr.us-east-1.amazonaws.com/amazonlinux:latest)
+  ContainerImage: "137112412989.dkr.ecr.us-east-1.amazonaws.com/amazonlinux:latest"
```

Comment:
Can this be changed?

Reply:
Yes, you can override that with a different ecr image.

Reply:
Yes, you can override with a different ecr image